components/datadog/sources/new-monitor-event/new-monitor-event.mjs

```diff
@@ -1,5 +1,5 @@
 import datadog from "../../datadog.app.mjs";
-import { payloadFormat } from "../common/common.mjs";
+import { payloadFormat } from "../common/payloadFormat.mjs";
```

Comment:
I think the convention in this repo is to name files using dash-case rather than camelCase. (`payloadFormat.mjs` -> `payload-format.mjs`)

Reply:
Addressed